src/EditorFeatures/Core.Cocoa/Microsoft.CodeAnalysis.EditorFeatures.Cocoa.csproj

```diff
@@ -10,6 +10,7 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <Compile Include="..\Core.Wpf\ExternalAccess\**\*.cs" LinkBase="ExternalAccess" />
```

Comment:
Can we limit it to `ExternalAccess\VSTypeScript\**\*.cs`?

Reply:
Can't hurt I guess.